cpp/include/cudf/rolling.hpp

```diff
@@ -275,5 +275,133 @@ std::unique_ptr<column> rolling_window(
   std::unique_ptr<aggregation> const& agg,
   rmm::mr::device_memory_resource* mr = rmm::mr::get_current_device_resource());
 
+/**
+ * @brief  Applies an offset-based window function (like LEAD/LAG) on specified column.
+ *
+ * Offset-based window functions include LEAD and LAG:
+ *   1. LEAD(N): Returns the row from the input column, at the specified offset past the
+ *      current row. If the offset crosses the grouping boundary or column boundary for
+ *      a given row, a "default" value is returned if available. The "default" value is
+ *      null, by default.
+ *   2. LAG(N): returns the row from the input column at the specified offset preceding
+ *      the current row. If the offset crosses the grouping boundary or column boundary for
+ *      a given row, a "default" value is returned if available. The "default" value is
```

Comment:
If the default value is null by default, do we need to state "if available"? It seems like there will always be a default value available.

Reply:
I can reword this.

Reply:
Hey, @hyperbolic2346. I have accommodated your suggestion at #6277. Please note that in that approach, I've had to move this documentation into `rolling.cu`, and removed `offset_rolling_window()` entirely.